build.sbt

```diff
@@ -70,16 +66,16 @@ licenses += ("BSD", url("http://opensource.org/licenses/BSD-3-Clause"))
 
 bintrayReleaseOnPublish := ! isSnapshot.value
 
-publishTo := {
+publishTo := Def.taskDyn[Option[Resolver]] {
```

Comment:
sbt 1.1 throws an error if the conditional isn't wrapped in a dynamic task:

```
cloudformation-template-generator/build.sbt:73: error: The evaluation of `publishTo` happens always inside a regular task.

Problem: `publishTo` is inside the if expression of a regular task.
  Regular tasks always evaluate task inside the bodies of if expressions.

Solution:
  1. If you only want to evaluate it when the if predicate is true or false, use a dynamic task.
  2. Otherwise, make the static evaluation explicit by evaluating `publishTo` outside the if expression.

    publishTo.value /* Value set by bintray-sbt plugin */
    ^
[error] Type error in expression
```